setup.rst

```diff
@@ -166,7 +166,9 @@ Once ``configure`` is done, you can then compile CPython with:
 This will build CPython with only warnings and errors being printed to
 stderr and utilize up to 2 CPU cores. If you are using a multi-core machine
 with more than 2 cores (or a single-core machine), you can adjust the number
-passed into the ``-j`` flag to match the number of cores you have.
+passed into the ``-j`` flag to match the number of cores you have (or if your
+version of Make supports it, you can leave the number off and Make will use
+as many cores as your machine has).
```

Comment:
> and Make will use as many cores as your machine has

I don't think that's quite right.  For example, on macOS 10.14, the current `man make` (which is for `GNU Make 3.81`) states:

> If the -j option is given without an argument, make will not limit the  number of jobs that can run simultaneously.

and, looking at the output of `make -d` leads me to believe that the number is indeed not related to the number of cores but can be much more.